src/createStore.js

```diff
@@ -71,6 +71,14 @@ export default function createStore(reducer, initialState, enhancer) {
    * @returns {any} The current state tree of your application.
    */
   function getState() {
+    if (isDispatching) {
+      throw new Error(
+        'You may not call store.getState() while the reducer is executing.' +
```

Comment:
These should have spaces right after the period, or you’ll end up with `executing.The reducer`